Documentation/ceph-disaster-recovery.md

```diff
@@ -81,18 +81,18 @@ ceph-mon \
     --log-stderr-prefix=debug \
     --default-log-to-file=false \
     --default-mon-cluster-log-to-file=false \
-    --mon-host=$(ROOK_CEPH_MON_HOST) \
-    --mon-initial-members=$(ROOK_CEPH_MON_INITIAL_MEMBERS) \
+    --mon-host=$ROOK_CEPH_MON_HOST \
+    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
```

Comment:
Where are those variables populated?

Reply:
I encountered this while executing ceph-mon command in the mon-b pod. 
kubectl -n rook-ceph exec -it <mon-pod> bash
--
[root@rook-ceph-mon-b-5b5dc6ff95-s4789 /]# echo $(ROOK_CEPH_MON_INITIAL_MEMBERS)
bash: ROOK_CEPH_MON_INITIAL_MEMBERS: command not found

[root@rook-ceph-mon-b-5b5dc6ff95-s4789 /]# echo $ROOK_CEPH_MON_INITIAL_MEMBERS 
a,b

Reply:
Ok, the pod is running, never mind then.